### DISCUSSION TITLE
[Docs] Example snippets in docs

### DISCUSSION THREAD

**Discussion by njwest** [photonstorm/phaser#5910] [Feature Requests]
Many thanks for the incredible game engine, Phaser team! This software is a ton of fun to work with.

One huge enhancement that could be made for Phaser 3 beginners at the cost of some busywork would be adding example code snippets to [the docs](https://photonstorm.github.io/phaser3-docs/) for each method. While the docs do a good job of explaining argument and return types, function usage is still a mystery in many cases if users aren't already familiar with the objects involved.

I'd be happy to help with this if I had a higher level of Phaser expertise...

**Comment by photonstorm** [comment]
While this is a valid and sensible request, the under-taking required to do it is phenomenal. I feel this is better handled via a combination of us _and_ (the majority of it) via the community.

**Comment by njwest** [comment]
Thanks for the response, @photonstorm , your software is amazing :]

I'm a month into my Phaser journey, and @rexrainbow 's [Phaser notes](https://github.com/rexrainbow/phaser3-rex-notes/) have been phenomenally helpful as an addition to the official Phaser 3 examples/docs (I'm not alone in this regard; Rex's repo has 651 stars and 159 forks)

Rather than writing all new examples within Phaser's language docs, a more efficient approach for Phaser 3 may be to add  links to relevant examples from Rex's notes, as well as links to relevant official examples, to specific method docs
